satellite/audit/service.go

```diff
@@ -30,6 +31,13 @@ type Config struct {
 	MinBytesPerSecond  memory.Size   `help:"the minimum acceptable bytes that storage nodes can transfer per second to the satellite" default:"128B"`
 	MinDownloadTimeout time.Duration `help:"the minimum duration for downloading a share from storage nodes before timing out" default:"25s"`
 	MaxReverifyCount   int           `help:"limit above which we consider an audit is failed" default:"3"`
+
+	ReservoirConfig
+}
+
+// ReservoirConfig contains configurable values for audit reservoirs
+type ReservoirConfig struct {
+	Slots int `help:"number of reservoir slots allotted for nodes" default:"1"`
```

Comment:
Why not just have `ReservoirSlots` in the main `Config` struct above? Is this extra struct necessary?

Reply:
that's true! I'll move it